cli/smartcontract/testdata/invalid9/invalid.yml

```diff
@@ -0,0 +1,16 @@
+name: Test undeclared event
+events:
+  - name: SomeEvent
+    parameters:
+      - name: p1
+        type: Array
+        extendedtype:
+          base: Array
+          name: invalid9.NamedStruct
+namedtypes:
+  invalid9.NamedStruct:
```

Comment:
But it's OK to have a `NamedStruct` here?

Reply:
Yes.